### PR TITLE
Fix/another typo

### PR DESCRIPTION
**Pull request by MoKob**
# Issue

Followup on https://github.com/Project-OSRM/osrm-backend/issues/2850

## Tasklist
 - [x] add regression / cucumber cases (see docs/testing.md)
 - [x] review
 - [ ] adjust for for comments

### CODE REVIEW

File: features/guidance/turn-lanes.feature [hunk 1]
```diff
@@ -120,6 +120,26 @@ Feature: Turn Lane Guidance
...
+            | ab    | road | left\|through\|     | 1                 | yes    |
+            | bc    | road |                     |                   | yes    |
+            | bd    | turn |                     |                   | yes    |
+            | cf    | turn |                     |                   | yes    |
+            | ce    | side |                     |                   | yes    |
+
+        When I route I should get
+            | waypoints | route          | turns                   | lanes                       |
+            | a,d       | road,turn,turn | depart,turn left,arrive | ,left:true straight:false,  |
+            | a,c       | road,road      | depart,arrive           | ,                           |
```
Comment (daniel-j-h) [inline features/guidance/turn-lanes.feature]: There are no lanes here since going straight is obvious and we silence lanes for this? It should be still on the intersection array and at some point we have to test for this, correct?

Comment (MoKob) [inline features/guidance/turn-lanes.feature]: yes, but unrelated change, see https://github.com/Project-OSRM/osrm-backend/issues/2702
